tests/NLog.UnitTests/LayoutRenderers/Wrappers/CachedTests.cs

```diff
@@ -114,9 +114,17 @@ public void CacheKeyTest()
             var s3 = GetDebugLastMessage("debug");
             Assert.NotEmpty(s3);
             Assert.NotEqual(s1, s3);
-
         }
 
-
+        [Fact]
+        public void CachedSecondsTimeoutTest()
```

Comment:
this one is not for this PR, right?

Reply:
Not sure I understand. I created both layoutrenderer together because `NetworktIpAddressLayoutRenderer` has a huge overhead.

Reply:
My bad. I was confused by the PR title.